roles/k8s-internal-container-registry/tasks/main.yml

```diff
@@ -6,9 +6,11 @@
 
 - name: install stable helm repository
   command: "{{ helm_cmd }} repo add {{ container_registry_repo_name }} {{ container_registry_chart_location }}"
+  changed_when: false
```

Comment:
Why did you add this? This should be consistent with all the helm install roles.